utils/text_formatting.jsx

```diff
@@ -49,13 +49,15 @@ export function formatText(text, inputOptions) {
     }
 
     let output = text;
-
     const options = Object.assign({}, inputOptions);
+    const hasPhrases = (/"([^"]*)"/).test(options.searchTerm);
 
-    if (options.searchMatches) {
+    if (options.searchMatches && !hasPhrases) {
```

Comment:
I don't think it's too much of a problem, but this is going to break fuzzy highlighting when using phrases. For example, `"row boat"` will match `row boat` and `row boating`, but this change will cause it to only highlight `row boat`

Reply:
That's interesting. When I search `"row boat"` I assume it to only match that exact phrase and not a superset of it. Not sure why `ElasticSearch` behaves like that or if this is expected. I'll investigate further.